sbpy/data/tests/test_orbit.py

```diff
@@ -161,3 +162,96 @@ def test_oo_propagate(self):
+HALLEY = {
+    'targetname': '1P/Halley',
+    'M1': u.Quantity(5.5, 'mag'),
+    'M2': u.Quantity(13.6, 'mag'),
+    'k1': 8.0,
+    'k2': 5.0,
+    'phasecoeff': u.Quantity(0.03, 'mag / deg'),
+    'e': 0.9663712597171566,
+    'q': u.Quantity(0.60100362, 'au'),
+    'incl': u.Quantity(162.21355489, 'deg'),
+    'Omega': u.Quantity(59.06301115, 'deg'),
+    'w': u.Quantity(111.98929651, 'deg'),
+    'n': u.Quantity(0.01304531, 'deg / d'),
+    'M': u.Quantity(172.4302269, 'deg'),
+    'nu': u.Quantity(179.49619633, 'deg'),
+    'a': u.Quantity(17.87172549, 'au'),
+    'Q': u.Quantity(35.14244737, 'au'),
+    'P': u.Quantity(27596.12824787, 'd'),
+    'epoch': Time(2459696.54292208, scale='tdb', format='jd'),
+    'Tp': Time(2446478.74665706, scale='tdb', format='jd'),
+    'orbtype': 'KEP'
+}
+
+# 252P and 2016 BA14 retrieved from Horizons on 26 Apr 2022
+COMETS = {
+    'targetname': ['252P/LINEAR', 'PANSTARRS (P/2016 BA14)'],
+    'M1': u.Quantity([16.4, 21.3], 'mag'),
+    'e': [0.671813410097529, 0.6650394891797625],
+    'k1': [16.5, 4.5],
+    'q': u.Quantity([1.00097003, 1.01345464], 'au'),
+    'incl': u.Quantity([10.41045416, 18.89502003], 'deg'),
+    'Omega': u.Quantity([190.93492189, 180.51874455], 'deg'),
+    'w': u.Quantity([343.32963355, 351.92333244], 'deg'),
+    'n': u.Quantity([0.18503493, 0.18727854], 'deg / d'),
+    'M': u.Quantity([53.61784042, 58.76185268], 'deg'),
+    'nu': u.Quantity([133.73260496, 136.4433599], 'deg'),
+    'a': u.Quantity([3.05000283, 3.02559439], 'au'),
+    'Q': u.Quantity([5.09903563, 5.03773413], 'au'),
+    'P': u.Quantity([1945.5786428, 1922.27043206], 'd'),
+    'M2': u.Quantity([np.nan, 23.1], 'mag'),
+    'k2': [np.nan, 5.0],
+    'phasecoeff': u.Quantity([np.nan, 0.03], 'mag / deg'),
+    'epoch': Time([2459696.55177132, 2459696.55177132], scale='tdb',
+                  format='jd'),
+    'Tp': Time([2459406.78031243, 2459382.78462704], scale='tdb', format='jd'),
+    'orbtype': ['KEP', 'KEP']
+ }
+
+# Jupiter retrieved from Horizons on 26 Apr 2022
+JUPITER = {
+    'targetname': 'Jupiter (599)',
+    'e': 0.04833017971265399,
+    'q': u.Quantity(4.95081216, 'au'),
+    'incl': u.Quantity(1.3036833, 'deg'),
+    'Omega': u.Quantity(100.52981374, 'deg'),
+    'w': u.Quantity(273.5645208, 'deg'),
+    'n': u.Quantity(0.08310479, 'deg / d'),
+    'M': u.Quantity(337.58379998, 'deg'),
+    'nu': u.Quantity(335.34786485, 'deg'),
+    'a': u.Quantity(5.20223722, 'au'),
+    'Q': u.Quantity(5.45366228, 'au'),
+    'P': u.Quantity(4331.8804878, 'd'),
+    'epoch': Time(2459696.54405449, scale='tdb', format='jd'),
+    'Tp': Time(2459966.27821974, scale='tdb', format='jd'),
+    'orbtype': 'KEP'
+}
+
+
+class TestOrbitTisserandDCriterion:
+    def testTisserand(self):
+        """ test tisserand method"""
+
+        halley = Orbit.from_dict(HALLEY)
+        jupiter = Orbit.from_dict(JUPITER)
+        assert u.isclose(halley.tisserand(jupiter), u.Quantity(-0.61659744))
```

Comment:
JPL SBDB has -0.605, which we can reproduce if we use the same epoch as their orbital elements:
```python
>>> epoch = Time(2449400.5, format='jd')
>>> halley = Orbit.from_horizons('1P', id_type='designation', closest_apparition=True, epochs=epoch)
>>> jupiter = Orbit.from_horizons(599, id_type=None, epochs=epoch)
>>> halley.tisserand(jupiter)
Out[47]: <MaskedQuantity -0.60495016>
```

Reply:
I will add this test to test_orbit_remote.py.